src/main/resources/containers.json

```diff
@@ -1250,8 +1250,6 @@
     ]
   },
   {
-    "artifact_id": "glassfish",
-    "group_id": "org.glassfish.main.distributions",
     "name": "Arquillian Container GlassFish Managed",
```

Comment:
Totally agree. In the long run however, it would be great to move this config metadata closer to the code (e.g. even in a form of annotations on the class itself). We should think about it.